community/public/css/style.less

```diff
@@ -1,14 +1,21 @@
 @primary-color: #08B74F;
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
I prefer not to have external dependencies. Could you please add the css and the fonts to `public/` folder?